gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -10,15 +10,20 @@ class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
     DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    LOG_FILE = '/var/www/miq/vmdb/log/ha_admin.log'.freeze
     attr_accessor :failover_attempts, :db_check_frequency, :failover_check_frequency
 
     def initialize(db_yml_file: '/var/www/miq/vmdb/config/database.yml',
                    failover_yml_file: '/var/www/miq/vmdb/config/failover_databases.yml',
                    ha_admin_yml_file: '/var/www/miq/vmdb/config/ha_admin.yml',
-                   log_file: '/var/www/miq/vmdb/log/ha_admin.log',
+                   logger: nil,
```

Comment:
Also, we're currently using all defaults when running this as a daemon on the appliance

https://github.com/manageiq/manageiq-appliance/blob/master/LINK/usr/bin/evm_failover_monitor.rb#L6